THANKS.txt

```diff
@@ -171,3 +171,4 @@ Agilent which gave a genereous donation for support of SciPy.
 UC Berkeley for providing travel money and hosting numerous sprints.
 The University of Stellenbosch for funding the development of
     the SciKits portal.
+Google Inc.
```

Comment:
It's a little delicate---because I work there, the company owns the copyright to any contributions I make (including on my own time) and needs to be acknowledged among the contributors.  Would something like "for supporting work on the documentation" be good?